crt_portal/cts_forms/templates/forms/errors.html

```diff
@@ -3,17 +3,17 @@
 {% load static %}
 
 {% block page_header %}
-<header class="confirmation-header">
+<header class="confirmation-header crt-landing--blue padding-bottom-10">
   <div class="grid-container">
     <div class="grid-row grid-gap flex-child">
       <div class="tablet:grid-col-9 tablet:grid-offset-1">
         <div class="doj-brand display-flex flex-align-center font-serif-lg">
-          <img src="{% static "img/doj-logo-footer.svg" %}" alt="" height="38" class="margin-right-105" />
-          <span>United States Department of Justice<span>
+          <img src="{% static "img/doj-logo-footer.svg" %}" alt="" height="55" class="margin-right-105" />
```

Comment:
We do need to fill out the alt tag here. Surprised our a11y tests don't flag this. (to be clear, this isn't your fault, but since we're updating this part, let's toss in a description while we're at it)